src/layout/Header/Navbar/index.jsx

```diff
@@ -222,12 +224,32 @@ const list = [
 ];
 
 const Navbar = () => {
-  const fullName = 'Tuan';
-  const total = 10;
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { token, userInfo } = useSelector((state) => state.userReducer);
+  const { cart } = useSelector((state) => state.cartReducer);
 
-  const handleLogout = () => {};
-  const renderCart = () => {};
+  const handleLogout = () => {
+    dispatch(logoutAction());
+  };
+  const renderCart = () => {
+    return cart.map((item, index) => {
+      return (
```

Comment:
đã dùng arrow function thì nên lược bỏ bớt keyword return nếu thật sự k cần thiết nhé
```suggestion
  const renderCart = () => cart.map((item, index) => (
```